src/ui/components/DevToolsDynamicLoadingMessage.tsx

```diff
@@ -69,7 +69,15 @@ const FILLER_PHRASES = [
   "Compiling tachyonic antitelephone scripts...",
 ];
 
-const LONG_WAIT_MESSAGE =
-  "<div><p>This is taking longer than expected.</p><p><a href='http://replay.io/discord' target='discord'>Contact us on Discord</a></p>";
+const LONG_WAIT_MESSAGE = (
+  <div>
+    <p>This is taking longer than expected.</p>
+    <p>
+      <a href="http://replay.io/discord" target="discord">
+        Contact us on Discord
+      </a>
+    </p>
+  </div>
+);
```

Comment:
This was a regression from a couple of weeks ago that I just happened to notice.